### DISCUSSION TITLE
IIP-2: Change the time of validation ceremony

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#153] [IIPs]
Change the time of validation ceremony from 13:30 UTC to 15:00 UTC

Read more details [here](https://docs.idena.io/docs/iip/iip-2).

**Comment by teavar** [comment]
I don't think that's a good idea, most people start or finish working at hh:00. imho a more inclusive/correct time would be hh:30

**Comment by teavar** [comment]
also I think that iips should be voted on separately, not bundled up like in the upcoming jarrett fork
